web/modules/custom/custom_services/src/ShoppingCart.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\custom_services;
+
+use Drupal\custom_services\Event;
+class ShoppingCart {
+
+  private $shopping_cart = NULL;
```

Comment:
missing comments above the parameters, you should keep 1 empty line between them